storm-client/test/jvm/org/apache/storm/utils/UtilsTest.java

```diff
@@ -173,4 +176,49 @@ public void isZkAuthenticationConfiguredStormServerWithPropertyTest() {
             }
         }
     }
+
+    @Test
```

Comment:
Can we have some tests for isValidConf in addition to checkMapEquality?

Reply:
Found a positive test case in TestConfigValidate and added negative test cases. Should I collect some more test cases?